ceph-pull-requests/config/definitions/ceph-pull-requests.yml

```diff
@@ -0,0 +1,64 @@
+- job:
+    name: ceph-pull-requests
+    project-type: freestyle
+    defaults: global
+    display-name: 'ceph: Pull Requests'
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - github:
+          url: https://github.com/ceph/ceph/
+    logrotate:
+      daysToKeep: 15
+      numToKeep: 30
+      artifactDaysToKeep: -1
+      artifactNumToKeep: -1
+
+    parameters:
+      - string:
+          name: sha1
+          description: "A pull request ID, like 'origin/pr/72/head'"
+
+    triggers:
+      - github-pull-request:
+          admin-list:
+            - alfredodeza
+            - ktdreyer
+            - andrewschoen
+            - dmick
+          org-list:
+            - ceph
```

Comment:
IIRC "org" never worked with our old Jenkins setup, but it would be awesome if it's finally working now.

Otherwise I suspect we'll want to expand "white-list" here.